mixins/arrow-keys-mixin.js

```diff
@@ -24,7 +24,7 @@ export const ArrowKeysMixin = superclass => class extends superclass {
 	}
 
 	arrowKeysContainer(inner) {
-		return html`<div class="arrow-keys-container" @keyup="${this._handleArrowKeys}">
+		return html`<div class="arrow-keys-container" @keyup="${this._handleArrowKeys}" @keydown="${this._handleKeyDown}">
```

Comment:
@dlockhart : I had Jeff switch this to use `keyup` as part of the move to the mixin since I thought it would be better for a11y.  It turns out the original reason for it using `keydown` was so it could prevent the default scrolling behavior that happens with arrow keys.  If we're preventing the default scrolling on `keydown` (as Jeff is proposing here), do you think we should even bother with `keyup`?

Reply:
Probably not worth the extra complexity, no.